README.md

```diff
@@ -1,2 +1,11 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+A) The task the algorithim is preforming is seeing if any elements are duplicated
+B) the worst case scenario would be checking all the elements in list I and J and only finding one duplicate at the end
```

Comment:
How does this lead to O(n^2) time complexity? Be specific -- e.g. for each element in the list, every other element must be checked